ghost/i18n/locales/lv/portal.json

```diff
@@ -0,0 +1,211 @@
+    "In your email client add {{senderEmail}} to your contacts list. This signals to your mail provider that emails sent from this address should be trusted.": "Savā e-pasta klientā pievienojiet {{senderEmail}} savam kontaktpersonu sarakstam. Tas norāda jūsu pasta pakalpojumu sniedzējam, ka no šīs adreses nosūtītajiem e-pasta ziņojumiem ir jābūt uzticamiem.",
+    "Invalid email address": "Nederīga e-pasta adrese",
+    "Jamie Larson": "Džeimijs Larsons",
+    "jamie@example.com": "jamie@example.com",
+    "Less like this": "Mazāk kā šis",
+    "Make sure emails aren't accidentally ending up in the Spam or Promotions folders of your inbox. If they are, click on \"Mark as not spam\" and/or \"Move to inbox\".": "Pārliecinieties, vai e-pasta ziņojumi nejauši nenonāk jūsu iesūtnes mapē Mēstules vai Reklāmas. Ja tie ir, noklikšķiniet uz \"Atzīmēt kā ne surogātpastu\" un/vai \"Pārvietot uz iesūtni\".",
+    "Manage": "Pārvaldīt",
+    "Maybe later": "Varbūt vēlāk",
+    "Memberships unavailable, contact the owner for access.": "Dalība nav pieejama. Lai iegūtu piekļuvi, sazinieties ar īpašnieku.",
+    "month": "mēnesis",
+    "Monthly": "Ikmēneša",
+    "More like this": "Vairāk kā šis",
+    "Name": "Vārds",
+    "Need more help? Contact support": "Vai nepieciešama papildu palīdzība? Sazinieties ar atbalsta dienestu",
+    "Newsletters can be disabled on your account for two reasons: A previous email was marked as spam, or attempting to send an email resulted in a permanent failure (bounce).": "Informatīvos izdevumus jūsu kontā var atspējot divu iemeslu dēļ: iepriekšējais e-pasts tika atzīmēts kā mēstule vai mēģinājums nosūtīt e-pastu izraisīja neatgriezenisku kļūmi (bounce).",
+    "No member exists with this e-mail address.": "Nav neviena dalībnieka ar šo e-pasta adresi.",
+    "No member exists with this e-mail address. Please sign up first.": "Nav neviena dalībnieka ar šo e-pasta adresi. Lūdzu, vispirms pierakstieties.",
+    "Not receiving emails?": "Vai nesaņemat e-pastus?",
+    "Now check your email!": "Tagad pārbaudiet savu e-pastu!",
+    "Once resubscribed, if you still don't see emails in your inbox, check your spam folder. Some inbox providers keep a record of previous spam complaints and will continue to flag emails. If this happens, mark the latest newsletter as 'Not spam' to move it back to your primary inbox.": "Ja pēc atkārtotas abonēšanas iesūtnē joprojām neredzat e-pasta ziņojumus, pārbaudiet surogātpasta mapi. Daži iesūtnes pakalpojumu sniedzēji reģistrē iepriekšējās surogātpasta sūdzības un turpinās atzīmēt e-pastus. Ja tā notiek, atzīmējiet jaunāko biļetenu kā “Nav mēstules”, lai pārvietotu to atpakaļ uz savu primāro iesūtni.",
+    "Permanent failure (bounce)": "Pastāvīga kļūme (bounce)",
+    "Phone number": "Tālruņa numurs",
+    "Plan": "Plānot",

```

Comment:
This should be a noun, not a verb.  It's a label for a button to select a subscription plan.  Should it be Plāns ?